### PR TITLE
l2s default speeds and polling times

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-1984
Fixes RNBW-1985

## What changed (plus any additional context for devs)

- Specific network gas polling times
- Arbitrum and optimism default to NORMAL speed. In order to do this I had to update `parseL2GasPrices` since I was using the average price before.
- Changed `parseGasPricesEtherscan` to `parseL2GasPrices` just to be clear we don't user etherscan
- removed parse eth gas station
- 
## Final checklist
[x] Assigned individual reviewers?
[x] Added labels?

### CODE REVIEW

File: src/utils/gas.ts [hunk 1]
```diff
@@ -10,7 +10,6 @@ const GasSpeedOrder = [NORMAL, FAST, URGENT, CUSTOM];
 
 const GAS_PRICE_SOURCES = {
   ARBITRUM_NODE: 'arbitrumNode',
-  ETH_GAS_STATION: 'ethGasStation',
   ETHERSCAN: 'etherscan',
```
Comment (brunobar79) [inline src/utils/gas.ts:13]: Should we rename this to something more accurate, like L2_NODE ?

**Review by brunobar79 — APPROVED** [review]
LGTM!

**Review by jinchung — APPROVED** [review]
🌮

File: src/parsers/gas.ts [hunk 2]
```diff
@@ -193,16 +194,18 @@ const parseGasPricesPolygonGasStation = (data: GasPricesAPIData) => {
...
+  network: Network
 ) => {
   if (!data) return getFallbackGasPrices();
-  switch (source) {
-    case gasUtils.GAS_PRICE_SOURCES.POLYGON_GAS_STATION:
+  switch (network) {
+    case Network.polygon:
       return parseGasPricesPolygonGasStation(data);
+    case Network.arbitrum:
+    case Network.optimism:
```
Comment (estebanmino) [inline src/parsers/gas.ts:208]: 👌

**Comment by jinchung** [comment]
Bruno has reapproved

**Pull request merged**